README.md

```diff
@@ -33,7 +35,7 @@ You'll find our [complete SDK documentation at https://customer.io/docs/sdk/reac
 
          dependencies {
             // Add this line:
-            classpath 'com.google.gms:google-services:4.3.13'  // Google Services plugin
+            classpath 'com.google.gms:google-services:<version-here>'  // Google Services plugin
```

Comment:
RN customers may not know how to find out what version to put in here. 

Is there a webpage or something we can help point them to to find this value?

Reply:
Agreed, I have already included [this link](https://developers.google.com/android/guides/google-services-plugin) at the start of this point. It should be easier to find preview mode.

Reply:
Ah! Sorry, I didn't see that link before. Thanks for pointing that out.